backend/src/controllers/clientCourses/sendAssessmentResult.ts

```diff
@@ -0,0 +1,27 @@
+import { NextFunction, Request, Response } from 'express';
+
+import CourseStatus from 'enums/coursesEnums';
+
+const sendAssessmentResult = async (
```

Comment:
maybe you can use adapterSendet.ts  that also send result to client

Reply:
fixed